WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Container/AddProductWithAIContainerViewModel.swift

```diff
@@ -23,6 +24,7 @@ final class AddProductWithAIContainerViewModel: ObservableObject {
 
     let siteID: Int64
     let source: AddProductCoordinator.Source
+    let featureFlagService: FeatureFlagService
```

Comment:
Super nit: is this for use later or redundant? I don't see it being used in this file.

Reply:
Thanks. I have started using this `featureFlagService ` now.